Header header
int32 laserCloudSurfFromMapNum
int32 laserCloudCornerFromMapNum
int32 laserCloudSurfStackNum
int32 laserCloudCornerStackNum
float64 total_translation
float64 total_rotation
float64 translation_from_last
float64 rotation_from_last
float64 time_elapsed
float64 latency
int32 n_iterations 
float64 average_distance 
float64 uncertainty_x
float64 uncertainty_y
float64 uncertainty_z
float64 uncertainty_roll
float64 uncertainty_pitch
float64 uncertainty_yaw
int32 meanSquareDistEdgeInlierNum
int32 meanSquareDistEdgeOutlierNum
int32 fitQualityCoeffEdgeInlierNum
int32 fitQualityCoeffEdgeOutlierNum
int32 meanSquareDistPlaneInlierNum
int32 meanSquareDistPlaneOutlierNum
int32 fitQualityCoeffPlaneInlierNum
int32 fitQualityCoeffPlaneOutlierNum
int32  PlaneMatchSuccess
int32  PlaneNoEnoughNeighbor
int32  PlaneNeighborTooFar 
int32  PlaneBADPCAStructure
int32  PlaneInvalidNumerical
int32  PlaneMSETOOLARGE
int32  PlaneUnknown
int32  PredictionSource 
super_odometry_msgs/IterationStats[] iterations